Plugins/Public/base_plugin/ExportData.cpp

```diff
@@ -156,12 +156,35 @@ void ExportData::ToJSON()
 		}
 		pwds.close();
 
+		minijson::object_writer shop = pw.nested_object("shop");
+		int curr_item = 1;
+		for (map<UINT, MARKET_ITEM>::iterator i = base->market_items.begin(); i != base->market_items.end(); ++i, curr_item++)
+		{
+			try {
+				shop.write("quantity", i->second.quantity);
+				shop.write("price", i->second.price);
+				shop.write("min_stock", i->second.min_stock);
+				shop.write("max_stock", i->second.max_stock);
+				shop.write("exporting", i->second.is_exporting);
```

Comment:
can we rename 'exporrting' and `is_exporting` to something that makes more sense please

Reply:
renamed to `is_public`